pkg/agent/core/ngt/service/ngt.go

```diff
@@ -135,9 +135,19 @@ func New(cfg *config.NGT, opts ...Option) (nn NGT, err error) {
 		}
 	}
 
-	n.kvs = kvs.New()
 	n.dim = cfg.Dimension
 
+	n.kvs = kvs.New(func() []kvs.Option {
+		if cfg.KVSDB.Concurrency < 1 {
+			return []kvs.Option{
+				kvs.WithErrGroup(n.eg),
+			}
+		}
+		return  []kvs.Option{
+			kvs.WithConcurrency(cfg.KVSDB.Concurrency),
+		}
+	}()...)
```

Comment:
I think we can do it like this.
```suggestion
	n.kvs = kvs.New(
	    kvs.WithConcurrency(cfg.KVSDB.Concurrency),
	)
```

Reply:
https://github.com/vdaas/vald/pull/1376#discussion_r688335874